packages/flutter_template_images/lib/flutter_template_images.dart

```diff
@@ -0,0 +1,3 @@
+library flutter_template_images;
+
+// There is no source for this package, since it only exists to hold images.
```

Comment:
i guess we need a dart file to make pub happy?

Reply:
I don't actually know; I decided to leave it like this because having no source felt like the kind of thing that might break later since it's such an odd thing to do. If you'd rather risk it I can try it out, it just didn't seem worth it to me. (Plus this makes things a bit more self-documenting if someone looks at the source instead of the README.)